static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -479,29 +483,36 @@ export const getEpicTestsFromGoogleDoc = (): Promise<
                                   useLocalViewLog: true,
                               }
                             : {}),
+                        ...(rows.some(row =>
```

Comment:
ah, so if it's set on any variant then it's set for the whole test?

Reply:
yes! could be a way of doing these test-level params, if we make sure to label it clearly in the spreadsheet

Reply:
(it does of course have the potential to be confusing too)

Reply:
Yep, up till now we've be limited to putting test-level settings in the name, or moving it to the variant-level.
If it's documented then it'll be fine. Perhaps one day we can implement something more suitable than a google sheet